source/_integrations/opower.markdown

```diff
@@ -17,8 +17,9 @@ ha_integration_type: integration
 
 The Opower integration allows you to get energy information from utilities that use [Opower](https://www.oracle.com/industries/utilities/opower-energy-efficiency/).
 
-More than 175 utilities use Opower. Currently only the following ones are supported:
+More than 175 utilities use Opower. Currently only the following utilities are supported by this integration:
 
+- Consolidated Edison (ConEd)
```

Comment:
Add "Orange & Rockland Utilities (ORU)" below this similar to https://github.com/tronikos/opower/blob/main/README.md